packages/@react-spectrum/datepicker/src/utils.tsx

```diff
@@ -31,7 +31,7 @@ export function useFormatHelpText(props: Pick<SpectrumDatePickerBase<any>, 'desc
       return (
         formatter.formatToParts(new Date()).map((s, i) => {
           if (s.type === 'literal') {
-            return <span key={i}>{s.value}</span>;
+            return <span key={i}>{` ${s.value} `}</span>;
```

Comment:
noticed in chromatic when i ran it on main. just need to add some extra spacing since that's what we had beefore